README.md

```diff
@@ -52,6 +52,9 @@ return {
 - `showBlankVirtLine = false`
   Setting this option will mean that if a Virtual Line would be blank it wont be
   rendered
+- `highlightColor` is set as a highlight table. It can be a `link` field
+  that must be a valid highlight, see `:highlight`. It can also be a table that defines
```

Comment:
Wording here is a bit awkward. Maybe "If used, the `link` field must refer to a valid highlight" or something like that?

Reply:
I have reworded it, i think it is clearer